docs/cli.md

```diff
@@ -240,7 +244,7 @@ If the element is of JSON/YAML type array list and you want trestle to create a
 
 If you just want to split a file into all its constituent parts and the file does not contain a simple list of objects, you can still use `*` and the file will be split into all its non-trivial elements.  Thus if you split a catalog with `-e catalog.*` the result will be a new directory, `catalog`, containing files representing the large items, `back-matter.json, groups.json and metadata.json`, but there will still be a `catalog.json` file containing just the catalog's `uuid`.  Small items such as strings and dates cannot be split off and will remain in the original model file that is being split.
 
-Here are some examples.  Starting with a single catalog file, `my_catalog/catalog.json`, if we do `trestle split -f catalog.json -e 'catalog.*'` we end up with:
+Here are some examples.  Starting with a single catalog file, `my_catalog/catalog.json`, if I do `trestle split -f catalog.json -e 'catalog.*'` I end up with:
```

Comment:
We may need to check all the docs. to convert we to I if it is done elsewhere like this.

Reply:
checked it here.